cnn_framework/predictor_test.py

```diff
@@ -104,14 +104,14 @@ def test_load_parameters(self):
 
         self.assertAlmostEqual(gfp.W_inner.eval()[0][0][0], 1.000, 3)
         self.assertAlmostEqual(gfp.b_inner.eval()[0][0][0], 0.000, 3)
-        self.assertAlmostEqual(gfp.W_output.eval()[0][0][0], 0.003, 3)
-        self.assertAlmostEqual(gfp.b_output.eval()[0][0][0], -0.843, 3)
+        self.assertAlmostEqual(gfp.W_output.eval()[0][0][0], 0.040, 3)
```

Comment:
Same comment here. Please squash and/or drop.

Reply:
Done